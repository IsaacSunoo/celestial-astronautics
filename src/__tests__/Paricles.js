import React from 'react';
import { shallow } from 'enzyme';
import ParticlesContainer from '../components/Particles';

describe('Particles container', () => {
    let wrapper;
    window.fetch = jest.fn();
    beforeEach(() => {
        wrapper = shallow(<ParticlesContainer />);
    });

    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});