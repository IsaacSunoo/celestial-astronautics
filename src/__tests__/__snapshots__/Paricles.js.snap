// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Particles container should match snapshot 1`] = `
<e
  className="particles-container"
  height="100%"
  params={
    Object {
      "interactivity": Object {
        "detect_on": "window",
        "events": Object {
          "onclick": Object {
            "enable": false,
            "mode": "bubble",
          },
          "onhover": Object {
            "enable": true,
            "mode": "repulse",
          },
          "resize": true,
        },
        "modes": Object {
          "bubble": Object {
            "distance": 400,
            "duration": 2,
            "opacity": 8,
            "size": 40,
            "speed": 3,
          },
          "grab": Object {
            "distance": 400,
            "line_linked": Object {
              "opacity": 1,
            },
          },
          "push": Object {
            "particles_nb": 4,
          },
          "remove": Object {
            "particles_nb": 2,
          },
          "repulse": Object {
            "distance": 100,
            "duration": 0.4,
          },
        },
      },
      "particles": Object {
        "color": Object {
          "value": "#ffffff",
        },
        "line_linked": Object {
          "color": "#ffffff",
          "distance": 0,
          "enable": true,
          "opacity": 0.3687847739990702,
          "width": 0.6413648243462091,
        },
        "move": Object {
          "attract": Object {
            "enable": false,
            "rotateX": 600,
            "rotateY": 1200,
          },
          "bounce": false,
          "direction": "none",
          "enable": true,
          "out_mode": "out",
          "random": false,
          "speed": 6,
          "straight": false,
        },
        "number": Object {
          "density": Object {
            "enable": true,
            "value_area": 1803.4120608655228,
          },
          "value": 150,
        },
        "opacity": Object {
          "anim": Object {
            "enable": false,
            "opacity_min": 0.1,
            "speed": 1,
            "sync": false,
          },
          "random": false,
          "value": 0.4008530152163807,
        },
        "shape": Object {
          "image": Object {
            "height": 100,
            "src": "img/github.svg",
            "width": 100,
          },
          "polygon": Object {
            "nb_sides": 4,
          },
          "stroke": Object {
            "color": "#000000",
            "width": 2,
          },
          "type": "circle",
        },
        "size": Object {
          "anim": Object {
            "enable": false,
            "size_min": 0.1,
            "speed": 40,
            "sync": false,
          },
          "random": true,
          "value": 1.5,
        },
      },
      "retina_detect": true,
    }
  }
  style={Object {}}
  width="100%"
/>
`;
